pkg/cleanup.go

```diff
@@ -39,7 +39,7 @@ func ChaosAbort(testsDetails *types.TestDetails) error {
 	}
 	log.Info("[Abort]: Chaos Experiment Aborted !!!")
 	// waiting for engine verdict updation
-	time.Sleep(3 * time.Second)
+	time.Sleep(5 * time.Second)
```

Comment:
Wait and check the verdict of engine  continuously to be patched as Stopped for abort chaos